package.json

```diff
@@ -12,7 +12,10 @@
     "prettier-check": "prettier --check .",
     "prettier-fix": "prettier --write .",
     "test": "vitest",
-    "test:coverage": "vitest run --coverage"
+    "test:coverage": "vitest run --coverage",
+    "typecheck": "tsc",
```

Comment:
How about renaming it to 'ts:check' to maintain consistency

Reply:
The `:` symbol is used to describe variations, there isn't a variation in this case.
and I suppose `ts` would mean `typescript`, I don't think it's semantic.
There are scripts that is one-worded, I don't think `typecheck` is inconsistent with the convention.

Reply:
Sure, indeed, I think your suggestion is good!!!

How about we adjust it to `test-types`? We can reference some well-known open-source projects like [nextjs](https://github.com/vercel/next.js/blob/canary/package.json#L15).

Actually, there's no standard answer for package.json script, but just want to follow similar naming rules as much as possible.

Reply:
Personally I still like `"typecheck"` more, it's not really a test and we can avoid the "-".
One thing to keep in mind, it could cause confusion if we write actual type tests with `vitest`.
Just offering my opinion, we totally can use `"test-types"` to follow community's convention.

Reply:
Actually, I prefer using the format `$tool:$command` or `$tool-$command`, as it looks clearer. 

The example of $tool-$command -> [react prettier command](https://github.com/facebook/react/blob/main/package.json#L128-L129), [react test command](https://github.com/facebook/react/blob/main/package.json#L120-L125)
The example of $tool:$command -> [babel](https://github.com/babel/babel/blob/main/package.json#L17-L20).

Let's leave this decision to a third party person~~ because discussing this between just the two of us isn't very meaningful for a code review. Everyone has their own ideas and habits.

i'm also assign to @cbot918 and @hcw0915

Reply:
Sorry for the delay. Since we are currently aiming to complete phase 1, I would like to integrate it into CI.

However, it seems to be stuck here, and no one has responded yet. Additionally, this is not a blocker, so let's proceed with the dev environment for now~~